scoap3/modules/records/util.py

```diff
@@ -55,13 +55,9 @@ def get_first_journal(record):
 
 
 def get_first_arxiv(record):
-    arxiv_array = [
-        a['value'].split(':')[1]
-        for a in record.get('report_numbers', ())
-        if a['source'] == 'arXiv'
-    ]
-    if arxiv_array:
-        return arxiv_array[0]
+    for arxiv in record.get('arxiv_eprints', ()):
```

Comment:
for your metal safety, since for every field you go through all the values to check if they are empty, and if not you do something, I can suggest you to use the `get_value` function from here https://github.com/inspirehep/inspire-utils/blob/bf94a7ad05fac5925b625bfc2491c54b4cfb5044/inspire_utils/record.py#L33.
It will avoid a lot of checking and copy/paste, it's easy to use and well tested.

Small example here:
```python
>> from inspire_utils.record import get_value
>> data = {'eprints': [{'val':1}, {'value':2}, {'val': 3}, {'value': 4}]}
>> get_value(data, 'eprints.value[0]')
2
```